_component_design/buttons.md

```diff
@@ -14,203 +14,193 @@ last-modified: 2017-08-17
 
 {% include toc.html %}
 
-Buttons provide the mechanism for users to perform actions within a
-Helix-themed UI. Each button has a weight (Primary, Secondary, and Tertiary),
-a size (Regular, Large, Compact), as well as a type (Simple, Dropdown).
+## Introduction
+{: .hxSectionTitle}
 
-{% figure [caption:"The three types of buttons defined in the Helix design language"] [class:"image bg-light border col-md-12"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-types.svg)
-{% endfigure %}
-
-# Primary Buttons
+Buttons provide the mechanism for users to perform actions within a Helix UI. Each button has a weight (Primary, Secondary, and Tertiary), a size (Regular, Large, Compact), as well as a type (Simple, Dropdown).
 
 <div class="hxRow">
-{% column left:"hxCol" %}
--   Use the primary action button to prompt the user to take action.
 
--   In most instances, use the standard size button. Use the compact buttons
-    with space constraints such as within a table.
 
--   Large buttons should be used in mobile designs to accommodate the ideal
-    touch target size.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### When to use
+{: .hxSubSectionTitle}
+Use a button to launch a process or initiate an action. Do not use buttons for navigation.  When the user needs to get from one page to another, use a text link. 
+
+### Best practice
+{: .hxSubSectionTitle}
+
+- Button Text style should always be Title Case where the first letter of each word is capitalized.
+- Buttons should always use the labelling convention of “Noun Verb”, with no articles. For example, use “Create Server”, not “Create a Server”.
+- Keep your text labels as short as possible while also fully communicating what action the button will trigger.
+- When choosing which button weight to use, use a secondary button unless you want the properties of a primary or tertiary button.
+- When choosing which button size to use, consider the context. In most cases, use a standard button.
+- There should only be one primary button in a form.
+- The primary button should always represent an inherently apparent next step such as "Create Server", "Add Selected", "Confirm Deletion",  "Submit”, or “Next”.
+- If you have multiple buttons on a page, there can only be one primary button.
 
--   Use one primary button per page.
 
 {% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Primary button sizes"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-primary-sizes.svg){:width="100%"}
-{% endfigure %}
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Primary button states"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-primary-actions.svg){:width="100%"}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the button pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-hero.svg){:width="100%"}
```

Comment:
Does this image have a lot of dead space around the border, or is that just me? It's making it look a lot smaller if it does...

Reply:
I cut all the extra space surrounding it and moved the elements closer to each other. Hero image is clearer now.